config.json

```diff
@@ -1908,11 +1909,35 @@
         "nugetId": "Xamarin.GooglePlayServices.Measurement.Base",
         "dependencyOnly": true
       },
+      {
+        "groupId": "com.google.auto.value",
+        "artifactId": "auto-value-annotations",
+        "version": "1.9.0",
+        "nugetVersion": "1.9.0.1",
+        "nugetId": "Xamarin.Google.AutoValue.Annotations",
+        "dependencyOnly": true
+      },
+      {
+        "groupId": "com.google.code.gson",
+        "artifactId": "gson",
+        "version": "2.9.0",
+        "nugetVersion": "2.9.0.1",
+        "nugetId": "GoogleGson",
+        "dependencyOnly": true
+      },
+      {
+        "groupId": "com.google.firebase",
+        "artifactId": "firebase-analytics",
+        "version": "16.3.0",
+        "nugetVersion": "116.3.0",
+        "nugetId": "Xamarin.Firebase.Analytics",
+        "dependencyOnly": true
+      },
       {
         "groupId": "com.google.guava",
         "artifactId": "guava",
-        "version": "31.1-android",
```

Comment:
Package downgrade.

If this is required, please provide information about why it is required.  If not, we should use the most recent version.

Reply:
@moljac did you address this one?

Reply:
yes

with latest:

```
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605: Detected package downgrade: Xamarin.Google.Guava from 31.1.0.3 to 31.1.0. Reference the package directly from the project to select a different version.  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Grpc.Android -> Xamarin.Google.Guava (>= 31.1.0.3)  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Google.Guava (>= 31.1.0) [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605: Detected package downgrade: Xamarin.Google.Guava from 31.1.0.3 to 31.1.0. Reference the package directly from the project to select a different version.  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Grpc.OkHttp -> Xamarin.Google.Guava (>= 31.1.0.3)  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Google.Guava (>= 31.1.0) [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605: Detected package downgrade: Xamarin.Google.Guava from 31.1.0.3 to 31.1.0. Reference the package directly from the project to select a different version.  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Grpc.Protobuf.Lite -> Xamarin.Google.Guava (>= 31.1.0.3)  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Google.Guava (>= 31.1.0) [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605: Detected package downgrade: Xamarin.Google.Guava from 31.1.0.3 to 31.1.0. Reference the package directly from the project to select a different version.  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Grpc.Stub -> Xamarin.Google.Guava (>= 31.1.0.3)  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Google.Guava (>= 31.1.0) [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]

```

Reply:
https://github.com/xamarin/GooglePlayServicesComponents/blob/mu-20220215-split-fb/config.json#L1934-L1941

```
      {
        "groupId": "com.google.guava",
        "artifactId": "guava",
        "version": "31.1.0-android",
        "nugetVersion": "31.1.0.3",
        "nugetId": "Xamarin.Google.Guava",
        "dependencyOnly": true
      },
```

Latest nuget:

https://www.nuget.org/packages/Xamarin.Google.Guava/31.1.0.3